### PR TITLE
bug: additional checks for card editor entities array

### PR DESCRIPTION
**Pull request by flixlix**
This resolves my issue #275 
Adds Additional checks in the editor. Checks if entities is actually an array before trying to do anything.

### CODE REVIEW

File: .gitignore
Comment (lyricnz) [inline .gitignore:1]: Suggest not to include unrelated things in PR

File: src/patch/hui-card-element-editor.ts [hunk 1]
```diff
@@ -68,14 +68,12 @@ customElements.whenDefined("hui-dialog-edit-card").then(() => {
         this._cardModIcon.icon = "mdi:brush";
       }
 
-      const button = this.shadowRoot.querySelector(
-        "mwc-button[slot=secondaryAction]"
-      );
+      const button = this.shadowRoot.querySelector("mwc-button[slot=secondaryAction]");
```
Comment (lyricnz) [inline src/patch/hui-card-element-editor.ts]: Again this is unrelated to your PR objective

**Comment by flixlix** [comment]
Reverted the changes unrelated to the PR

**Comment by dzedler** [comment]
Can someone review and accept the PR, please? I am affected by the bug this causes.

**Comment by Yorkie71** [comment]
Hi @thomasloven may I ask what the current status is with this task please as it would be really useful to have this approved so that the UI becomes available in the excellent power flow card that @flixlix has created. Thanks.

**Comment by DanielWeeber** [comment]
Hey @thomasloven same here, please have a look at it

**Comment by zanac** [comment]
When will be merged? This is a great PR

**Comment by marksie1988** [comment]
Any news on getting this merged? this will also help me with a similar issue

**Comment by thomasloven** [comment]
I had to do some massaging offline to get this to merge.

It's merged in https://github.com/thomasloven/lovelace-card-mod/commit/e7405bf3b2b641bc290d304c18f72439436e5f6a

Thanks a lot! And I'm really sorry this has caused issues for you and that I've been so slow to respond.

**Comment by Yorkie71** [comment]
Thanks @thomasloven I have just updated and now I see the GUI and Power Card Plus. Super!

**Comment by Se7enair** [comment]
It looks like that the problem is back in 2023.12.0 :(
